src/controllers/fetchEmail.js

```diff
@@ -233,37 +233,33 @@ export class FetchController extends BaseAPIController {
             mongo_id,
             status
         } = req.params;
-        MailProvider.changeUnreadStatus(req.checkBody, req.body, req.getValidationResult())
-            .then(() => {
-                req.email.find({
-                    mongo_id: mongo_id
-                }, (err) => {
-                    if (err) {
+        req.email.find({
+            _id: mongo_id
+        }, (err) => {
+            if (err) {
+                next(new Error(err));
+            } else if (status == "true" || status == "false") {
```

Comment:
sir in this data comes from front end , and always in the form of string , that's why we are using string here